src/base64.h

```diff
@@ -11,37 +11,26 @@
 namespace node {
 //// Base 64 ////
 static inline constexpr size_t base64_encoded_size(size_t size) {
-  return ((size + 2 - ((size + 2) % 3)) / 3 * 4);
+  return ((size + 2) / 3 * 4);
 }
 
 // Doesn't check for padding at the end.  Can be 1-2 bytes over.
-static inline size_t base64_decoded_size_fast(size_t size) {
-  size_t remainder = size % 4;
-
-  size = (size / 4) * 3;
-  if (remainder) {
-    if (size == 0 && remainder == 1) {
-      // special case: 1-byte input cannot be decoded
-      size = 0;
-    } else {
-      // non-padded input, add 1 or 2 extra bytes
-      size += 1 + (remainder == 3);
-    }
-  }
-
-  return size;
+static inline constexpr size_t base64_decoded_size_fast(size_t size) {
+  // 1-byte input cannot be decoded
+  return size > 1 ? size / 4 * 3 + (size % 4 + 1) / 2 : 0;
```

Comment:
```suggestion
  return size > 1 ? (size / 4) * 3 + (size % 4 + 1) / 2 : 0;
```

Reply:
Thank you for your suggestion! I fix it.